web/api/v1/api_test.go

```diff
@@ -2170,6 +2171,75 @@ func TestRespondError(t *testing.T) {
 		t.Fatalf("Expected response \n%v\n but got \n%v\n", res, exp)
 	}
 }
+func TestParseTimeParam(t *testing.T) {
+	type resultType struct {
+		asTime  time.Time
+		asError func() error
+	}
+	ts, _ := parseTime("1582468023986")
```

Comment:
testOk ? not strictly needed but best practice.

Reply:
Not sure what do you mean but I guess is renaming ```ts``` to ```testOk```?

Reply:
```suggestion
	ts, err := parseTime("1582468023986")
	testutil.Ok(t, err)
```